modules/opensearch/includes/opensearch.search.inc

```diff
@@ -585,18 +569,18 @@ function opensearch_search_sort_options() {
   if ($default_sort === 'rank_frequency' && empty($custom_ranking)) {
     $options = array_merge($options, [
       'rank_title' => [
-        'label' => t('Best match on title'),
-        'label_short' => t('Title'),
+        'label' => t('Title', array(), array('context' => 'opensearch_ranking')),
+        'label_short' => t('Title', array(), array('context' => 'opensearch_ranking')),
         'sort' => new TingSearchSort('rank_title'),
       ],
       'rank_creator' => [
-        'label' => t('Best match on creator'),
-        'label_short' => t('Creator'),
+        'label' => t('Creator', array(), array('context' => 'opensearch_ranking')),
+        'label_short' => t('Creator', array(), array('context' => 'opensearch_ranking')),
         'sort' => new TingSearchSort('rank_creator'),
       ],
       'rank_subject' => [
-        'label' => t('Best match on subject'),
-        'label_short' => t('Subject'),
+        'label' => t('Subject', array(), array('context' => 'opensearch_ranking')),
```

Comment:
ditto

Reply:
ditto - lets leave the discussion in the original comment:
https://github.com/ding2/ding2/pull/1173#discussion_r237084759